gitPractice/1-gitHomework.js

```diff
@@ -0,0 +1,45 @@
+//? practice to undrestand difference between const and let.
+//const and let are variable, on the other hand const is better than let because const variable won't be reasigned however let could be reasigned.
+// In conclusion:
+// "Var" variable can be updated and re-declared.don't need initialized.
+// "let" variable can be updated but not re-declared. don't need initialized.
+// "const" variable neither be update and re-declared. needs initialized.
+
+//?practice git and read the codeyourfuture git syllabus.
+//I read the git from codeYourFuture syllabus and also I watched the https://www.youtube.com/watch?v=BCQHnlnPusY and https://www.youtube.com/watch?v=iv8rSLsi1xo
+// Now I know how I can make a repo and how I can commit and add more commit, eventhough we could do the same in VS, git and gitHub.
+// In conclusion I need more practice to pull and push and make changes on it to clear all the options.
+
+//? git push, git pull, what is stage area in git, what is remote repository in git
+//? What is differece between local repository and remote repo in github.
+//"gitPull": If we want to update our branchs with our team mate, we need to write gitpull in terminal to check what have changed, on the other hand we could use git adn github.
+//"StageArea": is a temporary place to put our files to the up. in source control We need to click on + for each files we feel it could be finished, we could press the +.
+//"gitPush": If we want to send our changes in branch, we can write gitpush in terminal and also we could use git or github to update our branches.
+//"remoteRepository" I saw https://www.youtube.com/watch?v=lR_hYwCAaH4&list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV&index=10
+
+
+
+//? git cheatsheet -> find it.
+//Git CLI cheatsheet. This is a terminal which we could write rather than use git. However we could see the changes on the git as well.
+//"Terminal":
+//Using the terminal navigate to a specified folder (using cd)
+//Move up a level in a folder directory (using cd ..)
+//Cancel a running task in the terminal (using ctrl+c)
+// I could not find Windows WSL. I know that it is for linux.
+
```

Comment:
@RominaBideh  it is better to have a seprate folder of separate file for your self and note these things down for yourself

Reply:
@ellietms Sure, I have to do that, because they are so tricky, and I have to write down, besides irritation is an important thing for me. Thank you dear Elli for your time.